pulpcore/cli/common/__init__.py

```diff
@@ -127,6 +127,7 @@ def config_options(command: Callable[..., Any]) -> Callable[..., Any]:
 @click.option(
     "--dry-run", is_flag=True, help=_("Trace commands without performing any unsafe HTTP calls")
 )
+@click.option("--follow-redirects/--no-follow-redirects", default=False)
```

Comment:
I debated about making this a config option but decided against it since I don't think we really want to support handling of redirects but if we find we do, we can add a config option later.